### PR TITLE
chore: adds `irn_batchFetchMessages`

### PR DESCRIPTION
**Pull request by ganchoradkov**
and bumps patch version

### CODE REVIEW

File: jsonrpc/ws-connection/test/index.test.ts [hunk 1]
```diff
@@ -100,7 +100,7 @@ describe("@walletconnect/jsonrpc-ws-connection", () => {
       await new Promise<void>(async (resolve) => {
         conn.once("close", (event: CloseEvent) => {
           chai.expect(event.code).to.equal(3000);
-          chai.expect(event.reason).to.equal("Authorization error: Project ID is missing");
+          chai.expect(event.reason).to.equal("Project ID is missing");
```
Comment (ganchoradkov) [inline jsonrpc/ws-connection/test/index.test.ts:103]: this fixes the ci

Comment (devceline) [reply jsonrpc/ws-connection/test/index.test.ts:103]: :pray:

File: relay/relay-api/src/jsonrpc.ts [hunk 2]
```diff
@@ -27,5 +29,6 @@ export const RELAY_JSONRPC: { [protocol: string]: RelayJsonRpc.Methods } = {
     subscription: "iridium_subscription",
     unsubscribe: "iridium_unsubscribe",
     batchUnsubscribe: "iridium_batchUnsubscribe",
+    batchFetchMessages: "ididium_batchFetchMessages",
```
Comment (devceline) [inline relay/relay-api/src/jsonrpc.ts:32]:
```suggestion
    batchFetchMessages: "iridium_batchFetchMessages",
```

**Review by devceline — COMMENTED** [review]
Looks good to me except the spelling error -  the way I understand it we just need the types here so we can use them in the monorepo

**Comment by ganchoradkov** [comment]
>the way I understand it we just need the types here so we can use them in the monorepo

correct 💯

**Review by bkrem — APPROVED** [review]

**Pull request merged**